NewsCrawlerTrigger/ContentCrawler.js

```diff
@@ -1,4 +1,16 @@
 const Mercury = require('@postlight/mercury-parser')
+const Entities = require('html-entities').AllHtmlEntities
+
+const getContent = content => {
+	const entities = new Entities()
+
+	const rejex = /(<([^>]+)>)/gi
+
+	return entities
+		.decode(content)
+		.replace(rejex, '')
```

Comment:
If you strip off all html tags, then it will be just a long sentence. we have to explore AMP format